docs/site/src/demos/badges/SimpleBadge.js

```diff
@@ -0,0 +1,39 @@
+// @flow weak
+
+import React from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import Badge from 'material-ui/Badge';
+import customPropTypes from 'material-ui/utils/customPropTypes';
+import IconButton from 'material-ui/IconButton';
+import NotificationIcon from 'material-ui/svg-icons/notification';
+
+const styleSheet = createStyleSheet('SimpleBadge', () => ({
+  badge: {
```

Comment:
Two points here:
1. That style doesn't have enough specificity to be applied. It's a dead style
You need to add the badge component into the `MUI_SHEET_ORDER`.
2. Users shouldn't have to add that style. It should be handled by the badge.